jenkins/job-configs/bootstrap-maintenance.yaml

```diff
@@ -92,10 +96,12 @@
         frequency: '@daily'
         job-name: maintenance-daily
         repo-name: 'k8s.io/test-infra'
+        jenkins-timeout: 1800
```

Comment:
I do not think this is a great name.

There are two timeouts, both of which are jenkins timeouts:
1) The timeout for a build of the parent job (which schedules agent jobs on each agent)
2) The timeout for a build of the agent job (which runs the cleanup script on a particular agent)

Jeff noted that once the job starts the agent job has multiple layers of timeout logic (in addition to the timeout set by jenkins there is also a timeout in bootstrap)

Reply:
okay I explicitly listed out 3 different timeout we need here